### PR TITLE
Updates after organization rename

### PR DESCRIPTION
**Pull request by pkliczewski**
This PR removes usage of old name (Parodos) and fixes links.

### CODE REVIEW

File: docs/konflux/release_operator_with_konflux.md [hunk 1]
```diff
@@ -50,26 +50,26 @@ oc get snapshots --sort-by .metadata.creationTimestamp -l pac.test.appstudio.ope
 
 Example:
 ```console
-helm-operator-v2wsd		False	Merge pull request #170 from parodos-dev/appstudio-orchestrator-operator-bundle
+helm-operator-v2wsd		False	Merge pull request #170 from rhdhorchestrator/appstudio-orchestrator-operator-bundle
```
Comment (pkliczewski) [inline docs/konflux/release_operator_with_konflux.md:53]: @jenniferubah @jordigilh is this OK?

File: docs/konflux/release_operator_with_konflux.md [hunk 2]
```diff
@@ -190,10 +190,10 @@ schema: olm.bundle
   ...
   ...
   fbc-v4-14-5p7m9	True	Merge pull request #81 from jordigilh/release/1.2.0-rc6
-  fbc-v4-14-jv6f8	True	Merge pull request #83 from parodos-dev/konflux/references/main
-  fbc-v4-14-dhxqb	True	Merge pull request #82 from parodos-dev/konflux/component-updates/operator-bundle
-  fbc-v4-14-bdx8p	True	Merge pull request #85 from parodos-dev/konflux/component-updates/operator-bundle
-  fbc-v4-14-hftq5	True	Merge pull request #84 from parodos-dev/konflux/references/main
+  fbc-v4-14-jv6f8	True	Merge pull request #83 from rhdhorchestrator/konflux/references/main
```
Comment (pkliczewski) [inline docs/konflux/release_operator_with_konflux.md:193]: @jenniferubah @jordigilh same here

File: docs/konflux/release_operator_with_konflux.md [hunk 3]
```diff
@@ -396,10 +396,10 @@ schema: olm.bundle
...
```
Comment (pkliczewski) [inline docs/konflux/release_operator_with_konflux.md:399]: @jenniferubah @jordigilh same here

File: bundle/manifests/orchestrator-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -176,7 +176,7 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Parodos introduces serverless asynchronous workflows to Backstage,
+    - description: The orchestrator introduces serverless asynchronous workflows to Backstage,
```
Comment (jordigilh) [inline bundle/manifests/orchestrator-operator.clusterserviceversion.yaml]: suggestion:
s/orchestrator/RHDH Orchestrator

in any case, in my opinion the initial letter should be capitalized as it refers to a unique identity and not a general object or term.

File: config/manifests/bases/orchestrator-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -33,7 +33,7 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Parodos introduces serverless asynchronous workflows to Backstage,
+    - description: The orchestrator introduces serverless asynchronous workflows to Backstage,
```
Comment (jordigilh) [inline config/manifests/bases/orchestrator-operator.clusterserviceversion.yaml]: Same as before with regards to the `orchestrator` word

File: docs/konflux/release_operator_with_konflux.md (hunk 2 above)
Comment (jordigilh) [reply docs/konflux/release_operator_with_konflux.md:193]: ack

File: helm-charts/orchestrator/Chart.yaml [hunk 6]
```diff
@@ -23,6 +23,6 @@ version: 1.3.0
 # It is recommended to use it with quotes.
 appVersion: "0.0.1"
 
-# This is an icon to Parodos project. In the future we might design a dedicated logo for the Orchestrator.
-# However, since the entire development is done under the Parodos organization, we'll use it.
+# This is an icon to The orchestrator project. In the future we might design a dedicated logo for the Orchestrator.
+# However, since the entire development is done under the orchestrator organization, we'll use it.
```
Comment (jordigilh) [inline helm-charts/orchestrator/Chart.yaml]: same here: capital `O` if we've agreed about it earlier on.

File: helm-charts/orchestrator/Chart.yaml [hunk 7]
```diff
@@ -23,6 +23,6 @@ version: 1.3.0
 # It is recommended to use it with quotes.
 appVersion: "0.0.1"
 
-# This is an icon to Parodos project. In the future we might design a dedicated logo for the Orchestrator.
-# However, since the entire development is done under the Parodos organization, we'll use it.
+# This is an icon to The orchestrator project. In the future we might design a dedicated logo for the Orchestrator.
```
Comment (jordigilh) [inline helm-charts/orchestrator/Chart.yaml]: capital 'O' if we've agreed on it earlier on.

File: helm-charts/orchestrator/templates/tekton-pipeline.yaml [hunk 8]
```diff
@@ -45,9 +45,9 @@ spec:
           workspace: ssh-creds
       params:
         - name: GIT_USER_NAME
-          value: Parodos Tekton Pipeline
+          value: The orchestrator Tekton Pipeline
```
Comment (jordigilh) [inline helm-charts/orchestrator/templates/tekton-pipeline.yaml]: capital 'O' if we've agreed earlier on.

File: helm-charts/orchestrator/templates/tekton-pipeline.yaml [hunk 9]
```diff
@@ -142,9 +142,9 @@ spec:
...
```
Comment (jordigilh) [inline helm-charts/orchestrator/templates/tekton-pipeline.yaml]: capital 'O' if we've agreed earlier on.

File: docs/konflux/release_operator_with_konflux.md (hunk 1 above)
Comment (jenniferubah) [reply docs/konflux/release_operator_with_konflux.md:53]: Yes this looks fine to me. Will need to update our git repository in our Konflux configuration. Note to myself.

Comment (jenniferubah) [reply docs/konflux/release_operator_with_konflux.md:53]: Actually, I can see a PR for those changes. Jordi is already working on it.

**Review by jenniferubah — APPROVED** [review]

File: docs/konflux/release_operator_with_konflux.md (hunk 3 above)
Comment (jordigilh) [reply docs/konflux/release_operator_with_konflux.md:399]: I think it will be fine once the MR I raised is merged.

**Comment by masayag** [comment]
As @jordigilh pointed out, we should stick to `The Orchestrator` (capital 'O')
This will be aligned with the website in which 99% uses that format (the remaining 1% will be fixed by https://github.com/rhdhorchestrator/rhdhorchestrator.github.io/pull/126)

**Comment by pkliczewski** [comment]
@jordigilh @masayag sure, will update. Thank you for the review

File: bundle/manifests/orchestrator-operator.clusterserviceversion.yaml (hunk 4 above)
Comment (pkliczewski) [reply bundle/manifests/orchestrator-operator.clusterserviceversion.yaml]: done

File: config/manifests/bases/orchestrator-operator.clusterserviceversion.yaml (hunk 5 above)
Comment (pkliczewski) [reply config/manifests/bases/orchestrator-operator.clusterserviceversion.yaml]: done

File: helm-charts/orchestrator/Chart.yaml (hunk 7 above)
Comment (pkliczewski) [reply helm-charts/orchestrator/Chart.yaml]: done

File: helm-charts/orchestrator/templates/tekton-pipeline.yaml (hunk 9 above)
Comment (pkliczewski) [reply helm-charts/orchestrator/templates/tekton-pipeline.yaml]: done

File: helm-charts/orchestrator/templates/tekton-pipeline.yaml (hunk 8 above)
Comment (pkliczewski) [reply helm-charts/orchestrator/templates/tekton-pipeline.yaml]: done

File: helm-charts/orchestrator/Chart.yaml (hunk 6 above)
Comment (pkliczewski) [reply helm-charts/orchestrator/Chart.yaml]: done

**Review by masayag — APPROVED** [review]

**Pull request merged**